frontend/src/static/css/style.scss

```diff
@@ -275,14 +275,18 @@ header{
             margin-top:0.1rem;
             color:mui-color("red");
 
+            .filetype {
+                margin-left:3rem;
+            }
+
             .radio-button--search{
                 float:left;
                 margin:1.5rem 2.0rem 0rem 0rem;
             }
 
             .radio-button--checkbox{
                 float:left;
-                margin:0.25rem 2.0rem 0rem 0rem;
+                margin:0.25rem 2.0rem 0rem -3rem;
```

Comment:
instead of setting such a margin, would something like ` top:50%;` fix this more gracefully?

Reply:
I am not so sure how to use `left:30%` in our case. Using it may cause the distance between the radio button and checkbox button not fixed?

Reply:
Oh sorry, for a moment I thought this was the fix for vertical alignment issue.

![image](https://user-images.githubusercontent.com/19277206/53968414-f1f3cb00-4131-11e9-9fcc-8eab794bf5df.png)

Anyway, it still seems to be flushed to the top. Perhaps try to make it vertically centered?

Reply:
Can you examine the effect now?

Reply:
I have made it more align.